{
  "comments": [
    {
      "key": {
        "uuid": "baa39b77_4b56d067",
        "filename": "devices/debian.py",
        "patchSetId": 5
      },
      "lineNbr": 820,
      "author": {
        "id": 1016147
      },
      "writtenOn": "2019-06-14T20:44:39Z",
      "side": 0,
      "message": "whats this?",
      "revId": "0b6caa981fdfc283c0a14ce923b23b65411f99ee",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}